Makefile

```
@@ -0,0 +1,94 @@
+#
+# See ./CONTRIBUTING.rst
+#
+
+VERSION=$(shell grep __version__ httpie/__init__.py)
+REQUIREMENTS="requirements-dev.txt"
+TAG="\n\n\033[0;32m\#\#\# "
+END=" \#\#\# \033[0m\n"
+
+
+all: test
```

checkmake (0.2.2)

[warning] Target "all" should be declared PHONY.

(phonydeclared)